code/modules/mob/living/death.dm

```diff
@@ -35,15 +35,32 @@
 	flick(flick_name, animate)
 
 /mob/living/death(gibbed)
-	eye_blind = max(eye_blind, 1)
+	unset_machine()
+	reset_perspective(null)
+	if(client)
+		if(damageoverlay)
+			damageoverlay.overlays.Cut()

```

Comment:
When you are dead and inside your corpse, you no longer see the red circles of the damage hud overlays. There's no reason to see them when inside your corpse since you do not see them when you ghost.